res/config.json

```diff
@@ -14,7 +14,8 @@
   "redis": {
     "enabled": false,
     "connection": null,
-    "options": null
+    "options": null,
+    "scope": null
```

Comment:
It's useful to add new properties to the config.json file. Even if set to null it helps to know that this setting exists.